src/Symfony/Component/Validator/Mapping/ClassMetadata.php

```diff
@@ -262,6 +279,10 @@ public function addConstraint(Constraint $constraint)
      * @param Constraint $constraint The constraint
      *
      * @return ClassMetadata This object
+     *
+     * @throws GroupDefinitionException When the "class name" group has
```

Comment:
I'd replace `"class name" group` by `default group`, since this is what `getDefaultGroup()` returns.